src/Lib/CachedChannelFactory.cs

```diff
@@ -30,6 +30,27 @@ internal class CachedChannelFactory
         /// </summary>
         private static Dictionary<string, Channel> cache = new Dictionary<string, Channel>();
 
+        /// <summary>
+        /// The grpc setting name for maximum message length that can be received.
```

Comment:
Nit: Should this be gRPC rather than grpc from a naming convention standpoint?

Reply:
Done.